docs/tutorial/part-one/index.md

```diff
@@ -4,54 +4,54 @@ typora-copy-images-to: ./
 
-## Using Gatsby starters
+## Korzystanie ze starterów Gatsby
 
-In [**tutorial part zero**](/tutorial/part-zero/), you created a new site based on the “hello world” starter using the following command:
+W [**części zerowej poradnika**](/tutorial/part-zero/), stworzyłeś nową stronę bazującą na starterze “hello world” używając komendy:
 
 ```shell
 gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world
 ```
 
-When creating a new Gatsby site, you can use the following command structure to create a new site based on any existing Gatsby starter:
+Podczas tworzenia nowej strony Gatsby, możesz użyć poniższej komendy, aby stworzyć nową stronę bazując na jakimkolwiek istniejącym już starterze Gatsby:
 
 ```shell
-gatsby new [SITE_DIRECTORY_NAME] [URL_OF_STARTER_GITHUB_REPO]
+gatsby new [NAZWA_FOLDERU_STRONY] [URL_REPO_STARTERA_NA_GITHUB]
 ```
 
-If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero.
+Jeśli pominiesz ostatni URL, Gatsby automatycznie wygeneruje stronę bazując na [**domyślnym starterze**](https://github.com/gatsbyjs/gatsby-starter-default). Dla tej sekcji, zostań przy stronie “Hello World” którą stworzyłeś wcześniej w części zerowej poradnika.
 
-### ✋ Open up the code
+### ✋ Otwórz kod
 
-In your code editor, open up the code generated for your “Hello World” site and take a look at the different directories and files contained in the ‘hello-world’ directory. It should look something like this:
+W edytorze kodu, otwórz kod wygenerowany dla twojej strony “Hello World” i przyjrzyj się folderom i plikom znajdującym się w folderze ‘hello-world’. Powinno to wyglądać mniej więcej tak:
 
-![Hello World project in VS Code](01-hello-world-vscode.png)
+![Hello World projekt w VS Code](01-hello-world-vscode.png)
 
-_Note: Again, the editor shown here is Visual Studio Code. If you’re using a different editor, it will look a little different._
+_Note: Edytor pokazany tutaj to Visual Studio Code. Jeśli korzystasz z innego edytora, może to wyglądać nieco inaczej._
 
-Let’s take a look at the code that powers the homepage.
+Rzućmy okiem na kod, który zasila stronę główną.
 
-> 💡 If you stopped your development server after running `gatsby develop` in the previous section, start it up again now — time to make some changes to the hello-world site!
+> 💡 Jeśli zatrzymałeś już swój serwer deweloperski bo uruchomieniu `gatsby develop` w sekcji poprzedniej, wystartuj go znowu — czas wprowadzić kilka zmian na stronie hello-world!
 
-## Familiarizing with Gatsby pages
+## Zapoznanie ze stronami Gatsby
```

Comment:
```suggestion
## Zapoznanie się ze stronami Gatsby
```

Reply:
dla mnie to 'się' jest tu troche zbędne, brzmi to dla Ciebie lepiej?

Reply:
Hmm w sumie może masz rację :)